src/main/java/com/github/darkroomdevs/checkdigit/validator/brazil/BankAgencyBBValidator.java

```diff
@@ -0,0 +1,38 @@
+package com.github.darkroomdevs.checkdigit.validator.brazil;
+
+import com.github.darkroomdevs.checkdigit.util.ModuloUtil;
+import com.github.darkroomdevs.checkdigit.validator.DigitValidator;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * This class checks whether a specific string represents a valid bank account number of Banco Santander.
+ * <p/>
+ * Reference: <a href="http://hostmarcasite.com.br/fluir/vendor/eduardokum/laravel-boleto/manuais/Regras%20Validacao%20Conta%20Corrente%20VI_EPS.pdf">Banco do Brasil.</a>
+ */
+public final class BankAgencyBBValidator implements DigitValidator {
```

Comment:
You must use the prefix `BankBranch` when referring to the bank branch number.

Reply:
Alright.